modules/configure-acs-kube-security-platform.adoc

```diff
@@ -3,7 +3,11 @@
 // * integration/integrate-using-email.adoc
 :_module-type: PROCEDURE
 [id="configure-acs-kube-security-platform"]
+<<<<<<< HEAD
```

Comment:
this shows up in the preview

Reply:
@kcarmichael08 Sorry. It helps to save the file before committing it!